[docs-only] feat: change the docs to reflect the new deployments (#9504)
owncloud/ocis: docs/ocis/deployment/continuous_deployment.md

We are continuously deploying the following deployment examples. Every example is deployed in two flavors:

- Master: reflects the current master branch state of oCIS and will be updated with every commit to master

> - Rolling: reflects the latest rolling release of oCISt and will be updated with every release

Reviewer: of oCIS

Author: fixed.

- Production: reflects the latest production release of oCIS and will be updated with every release

The configuration for the continuous deployment can be found in the [oCIS repository](https://github.com/owncloud/ocis/tree/master/deployments/continuous-deployment-config).

# oCIS with Web Office

> This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with Collabora Online as the office suite, traefik reverse proxy, cloudimporter and the inbucket mail catching server to showcase the full feature set of oCIS.

Reviewer 2: ```suggestion
This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with traefik reverse proxy, Collabora Online as the office suite, Tika for search and Cloudimporter to showcase the full feature set of oCIS. The Inbucket mail catching server or other services can be easily configured additionally.
```
I have checked the `ocis_full/.env` file to see what is default enabled.

Author: Our config is different

Author: I would keep it as it is.

Credentials:

- oCIS: see [default demo users]({{< ref "../getting-started#login-to-owncloud-web" >}})

## Master

- oCIS: [ocis.ocis.master.owncloud.works](https://ocis.ocis.master.owncloud.works)
- Mail: [mail.ocis.master.owncloud.works](https://mail.ocis.master.owncloud.works)

## Production Release

- oCIS: [ocis.ocis.production.owncloud.works](https://ocis.ocis.production.owncloud.works)

> - Mail: [mail.ocis.production.owncloud.works](https://mail.ocis.production.owncloud.works)

Reviewer: could we move it after ## Rolling Release to keep structure: 
Master
Rolling 
Production

## Rolling Release

- oCIS: [ocis.ocis.rolling.owncloud.works](https://ocis.ocis.rolling.owncloud.works)
- Mail: [mail.ocis.rolling.owncloud.works](https://mail.ocis.rolling.owncloud.works)

## Rolling Release with OnlyOffice

This example is based on the [ocis_full Example](ocis_full.md) and uses the default configuration with OnlyOffice as the office suite.

> - oCIS: [ocis.ocis-onlyoffice.rolling.owncloud.works](https://ocis.ocis-wopi.onlyoffice.owncloud.works)

Reviewer: wrong url: 
`[ocis.ocis-onlyoffice.rolling.owncloud.works](ocis.ocis-onlyoffice.rolling.owncloud.works)`

# oCIS and ownCloud Web with both most recent development versions

Credentials:

- oCIS: see [default demo users]({{< ref "../getting-started#login-to-owncloud-web" >}})

> ## Latest

Reviewer: master?

Author: I would not make it explicit.

  - username: admin
  - password: admin

## Rolling Release

- oCIS: [ocis.ocis-keycloak.rolling.owncloud.works](https://ocis.ocis-keycloak.rolling.owncloud.works)
- Keycloak admin access: [keycloak.ocis-keycloak.rolling.owncloud.works](https://keycloak.ocis-keycloak.rolling.owncloud.works)
- Keycloak account management: [keycloak.ocis-keycloak.rolling.owncloud.works/realms/oCIS/account/#/](https://keycloak.ocis-keycloak.rolling.owncloud.works/realms/oCIS/account/#/)

# oCIS with S3 storage backend (MinIO)

> This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses a configuration with Collabora Online as the office suite, traefik reverse proxy, cloudimporter and the inbucket mail catching server to showcase the full feature set of oCIS. In addition to that, we deployed a MinIO S3 storage backend. oCIS stores the data in the S3 server and the metadata on the local disk by using the `s3ng` storage driver.

Reviewer 2: ```suggestion
This deployment is based on our modular [ocis_full Example](ocis_full.md), see the [oCIS with Web Office](#ocis-with-web-office) description for the feature set. In addition to that, we deployed a MinIO S3 storage backend. oCIS stores the data in the S3 server and the metadata on the local disk by using the `s3ng` storage driver.
```
